cdm/core/src/test/java/ucar/nc2/ncml/TestEnhanceClassifier.java

```diff
@@ -112,4 +117,26 @@ public void testEnhanceClassifier_integers() throws IOException {
     }
 
   }
+
+  @Test
+  public void testEnhanceClassifier_classification() throws IOException {
+
+    try (NetcdfFile ncfile = NetcdfDatasets.openDataset(dataDir + "testAddToClassifier.ncml", true, null)) {
+
+      Variable Classify_Specsx = ncfile.findVariable("class_specs");
```

Comment:
Once implemented, this should probably test `Classifier classifier = Classifier.createFromVar(Classify_Specsx)'

Reply:
Implemented